apisix/plugins/opentelemetry.lua

```diff
@@ -333,6 +333,12 @@ function _M.rewrite(conf, api_ctx)
         kind = span_kind.server,
         attributes = attributes,
     })
+
+    local span_context = ctx.sp:context()
+    ngx.var.opentelemetry_context_traceparent = string.format("00-%s-%s-%02x",span_context.trace_id, span_context.span_id, span_context.trace_flags)
+    ngx.var.opentelemetry_trace_id = span_context.trace_id
+    ngx.var.opentelemetry_span_id = span_context.span_id
```

Comment:
We should add a bool field `set_ngx_var`(default false) in opentelemetry attr_schema, so user can enable/disable it, and keep backward compatibility.